### DISCUSSION TITLE
What is default model used for Open AI ?

### DISCUSSION THREAD

**Discussion by vpurandara** [sinaptik-ai/pandas-ai#361] [Q&A] [answered]
### 🐛 Describe the bug

which model are using for OpenAI like "text-ada-001" or anything. Is there a option to set higher models for OpenAI like "	
text-davinci-003"

**Comment by gventuri** [comment, marked as answer]
Hi @vpurandara the default model is `gpt-3.5-turbo`.

Here's a list of all the supported models:
```
[
        "gpt-4",
        "gpt-4-0613",
        "gpt-4-32k",
        "gpt-4-32k-0613",
        "gpt-3.5-turbo",
        "gpt-3.5-turbo-16k",
        "gpt-3.5-turbo-0613",
        "gpt-3.5-turbo-16k-0613",
]
```

You can use the model you prefer passing it when you instantiate the LLM, like this:

```
OpenAI(model="gpt-4")
```

**Comment by vpurandara** [comment]
@gventuri , thanks
